src/applications/vaos/services/mocks/index.js

```diff
@@ -243,21 +243,21 @@ const responses = {
   //   return res.json(errors);
   // },
 
-  // 'GET /vaos/v2/appointments/:id': (req, res) => {
-  //   const appointments = {
-  //     data: requestsV2.data.concat(confirmedV2.data).concat(mockAppts),
-  //   };
-  //   const appointment = appointments.data.find(
-  //     appt => appt.id === req.params.id,
-  //   );
-
-  //   if (appointment.start) {
-  //     appointment.future = moment(appointment.start).isAfter(moment());
-  //   }
-  //   return res.json({
-  //     data: appointment,
-  //   });
-  // },
+  'GET /vaos/v2/appointments/:id': (req, res) => {
+    const appointments = {
+      data: requestsV2.data.concat(confirmedV2.data).concat(mockAppts),
+    };
+    const appointment = appointments.data.find(
+      appt => appt.id === req.params.id,
+    );
+
+    if (appointment.start) {
+      appointment.future = moment(appointment.start).isAfter(moment());
+    }
+    return res.json({
+      data: appointment,
+    });
+  },
```

Comment:
I was wonder why. this was commented out. I enabled it locally.

Reply:
It was disabled for EPS work [at one point](https://github.com/department-of-veterans-affairs/vets-website/pull/36004#discussion_r2056943888) but they've switched to using the `'GET /vaos/v2/eps_appointments/:appointmentId'` endpoint so we can re-enable this I think.